src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TseAdmReplyService.java

```diff
@@ -198,28 +199,31 @@ public void sendAdmReplyEmails(String caseId, CaseData caseData) {
         }
     }
 
-    private void collectRespondents(CaseData caseData, List<TSEAdminEmailRecipientsData> emailsToSend) {
-        // if respondent only or both parties: send Respondents Reply Emails
-        if (RESPONDENT_ONLY.equals(caseData.getTseAdmReplySelectPartyNotify())
-            || BOTH_PARTIES.equals(caseData.getTseAdmReplySelectPartyNotify())) {
-            TSEAdminEmailRecipientsData respondentDetails;
-            for (RespondentSumTypeItem respondentSumTypeItem: caseData.getRespondentCollection()) {
-                if (respondentSumTypeItem.getValue().getRespondentEmail() != null) {
-                    respondentDetails =
-                        new TSEAdminEmailRecipientsData(
-                            tseAdminReplyRespondentTemplateId,
-                            respondentSumTypeItem.getValue().getRespondentEmail());
-
-                    if (isResponseRequired(caseData, RESPONDENT_TITLE)) {
-                        respondentDetails.setCustomisedText(RESPONSE_REQUIRED);
-                    } else {
-                        respondentDetails.setCustomisedText(RESPONSE_NOT_REQUIRED);
-                    }
-
-                    emailsToSend.add(respondentDetails);
-                }
-            }
+    /**
+     * Send notify emails to Respondents (or LR if they are assigned).
+     */
+    public void sendNotifyEmailsToRespondents(CaseDetails caseDetails) {
+        CaseData caseData = caseDetails.getCaseData();
+        if (CLAIMANT_ONLY.equals(caseData.getTseAdmReplySelectPartyNotify())) {
+            return;
+        }
+
+        String customisedText = isResponseRequired(caseData, RESPONDENT_TITLE)
+                ? RESPONSE_REQUIRED : RESPONSE_NOT_REQUIRED;
+
+        Map<String, String> personalisation = buildPersonalisation(caseData.getEthosCaseReference(),
+                caseDetails.getCaseId(), customisedText);
+
+        List<RespondentSumTypeItem> respondents = caseData.getRespondentCollection();
+        respondents.forEach(obj -> sendRespondentEmail(caseData, personalisation, obj.getValue()));
+    }
+
+    private void sendRespondentEmail(CaseData caseData, Map<String, String> emailData, RespondentSumType respondent) {
```

Comment:
should this not be sending emails to respondent reps?

Reply:
It's just the confusing name, sorry. If you check method NotificationHelper.getEmailAddressForRespondent, it is getting emails of respondent reps.